svelte.config.js

```diff
@@ -12,7 +12,7 @@ const config = {
     target: '#svelte',
     adapter: adapter(),
     prerender: {
-      /** @type {import('@sveltejs/kit').PrerenderErrorHandler} */
+      // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
Why? There should be a comment clarifying this.

Reply:
Good call; I almost forgot why that was there myself 😅
[Did it!](https://github.com/EthanThatOneKid/acmcsuf.com/pull/261/commits/1703ab0d05901e424c1d559c82f1c7cc6cc44a74)